source/imaer-shared/src/test/java/nl/overheid/aerius/shared/emissions/FarmlandEmissionsCalculatorTest.java

```diff
@@ -29,16 +30,15 @@
 
 import nl.overheid.aerius.shared.domain.Substance;
 import nl.overheid.aerius.shared.domain.v2.source.FarmlandEmissionSource;
-import nl.overheid.aerius.shared.domain.v2.source.farmland.FarmlandActivity;
-import nl.overheid.aerius.shared.domain.v2.source.farmland.FarmlandGrazingActivity;
-import nl.overheid.aerius.shared.domain.v2.source.farmland.FarmlandManureActivity;
-import nl.overheid.aerius.shared.domain.v2.source.farmland.FarmlandOrganicProcessesActivity;
+import nl.overheid.aerius.shared.domain.v2.source.farmland.AbstractFarmlandActivity;
+import nl.overheid.aerius.shared.domain.v2.source.farmland.CustomFarmlandActivity;
+import nl.overheid.aerius.shared.domain.v2.source.farmland.StandardFarmlandActivity;
 import nl.overheid.aerius.shared.exception.AeriusException;
 
 @ExtendWith(MockitoExtension.class)
 class FarmlandEmissionsCalculatorTest {
 
-  private static final String GRAZING_CATEGORY_CODE = "grazingCategoryCode";
+  private static final String PASTURE_CATEGORY_CODE = "pastureCategoryCode";
```

Comment:
could use `CATEGORY_CODE = "someCategoryCode"` to avoid the pasture vs grazing discussion here ;)

Reply:
grazing was in more places in this test, even though it's not relevant anymore. Its just custom and standard now.